common/utils.cpp

```diff
@@ -56,7 +56,7 @@ uint32_t guessFourcc(const char* fileName)
         }
     }
 
-    return YAMI_FOURCC_I420;
+    return 0;
```

Comment:
We treat i420 as default yuv output.
This will break yamidecode -i xx.264 -o aaa.yuv.